### PR TITLE
BF: Add SPACE and define keyboard buttons visually.

### LINKED ISSUE #204: Unable to enter a space in ConnectionPanel
When attempting to join my wifi, I went to go enter my wifi's password, but discovered I wasn't able to enter a space character into the input

### PR DESCRIPTION
**Pull request by thebeline**
In Network SSID Password: adds `SPACE` and visually defines the buttons (necessary to be able tell there is an option for space... 
 ;-)

Resolves: #204

### CODE REVIEW

**Comment by thebeline** [comment]
Built and validated

File: styles/z-bolt/style.css [hunk 1]
```diff
@@ -145,8 +145,9 @@ entry {
 
 button.keyboard {
 	font-size: 30px;
-	margin: 0px;
-	/* margin-bottom: 10px; */
+	margin: 2px;
+	background-color: #538fbe;
```
Comment (JeffB42) [inline styles/z-bolt/style.css:149]: I'm going to leave this as-as for now (the background color) and merge this PR, but please be aware that OctoScreen has a "white on black" style to it, and this deviates from that.  I think here it's OK b/c the Connection panel is seldom used and is kind of a "one-off" feature.

File: ui/ConnectionPanel.go [hunk 2]
```diff
@@ -22,7 +22,7 @@ var keyBoardChars = []byte{
 	'0', '1', '2', '3', '4', '5', '6', '7', '8', '9',
 	'!', '@', '#', '$', '%', '^', '&', '*', '(', ')', '_', '+', '-',
 	'=', '.', ',', '|', ':', ';', '/', '~', '`', '[', ']', '{', '}',
-	'±', '§', '\\',
+	'±', '§', '\\', ' ',
```
Comment (JeffB42) [inline ui/ConnectionPanel.go:25]: 👍

File: styles/z-bolt/style.css (hunk 1 above)
Comment (thebeline) [reply styles/z-bolt/style.css:149]: I'm sorry.  Truth be told, I just copied and pasted the style from another project, initially for testing, and it worked and I kept it.  That was bad form and I will try to adhere to the style of the app next time.

**Comment by JeffB42** [comment]
Actually, I like it, and I think that the Connection panel is a fringe/one-off panel anyway, so that it's OK.  I just mentioned they style thing to bring it up, and to be aware of.

**Comment by thebeline** [comment]
I appreciate the feedback.

**Pull request merged**